Index: index.php
===================================================================
diff --git a/index.php b/index.php
deleted file mode 100644
--- a/index.php	(revision b7c89f75f6c67ec454c01e69cebcd925a4a97d77)
+++ /dev/null	(revision b7c89f75f6c67ec454c01e69cebcd925a4a97d77)
@@ -1,4 +0,0 @@
-<a href="user/login-demo.html">Login</a>
-<a href="user/personal-profile/personal-profile-demo.html">个人空间</a>
-<?php
-echo "hello world";
\ No newline at end of file
diff --git a/Homepage.html b/index.html
rename from Homepage.html
rename to index.html
